code/modules/cargo/supply_packs/seeds.dm

```diff
@@ -64,3 +64,43 @@
 	name = "Poppy"
 	cost = 50
 	contains = /obj/item/neuFarm/seed/poppy
+
+/datum/supply_pack/seeds/plum
+	name = "Plum"
+	cost = 15
+	contains = /obj/item/neuFarm/seed/plum
+
+/datum/supply_pack/seeds/lemon
+	name = "Lemon"
+	cost = 30
+	contains = /obj/item/neuFarm/seed/lemon
+
+/datum/supply_pack/seeds/lime
+	name = "Lime"
+	cost = 30
+	contains = /obj/item/neuFarm/seed/lime
+
+/datum/supply_pack/seeds/tangerine
+	name = "Tangerine"
+	cost = 30
+	contains = /obj/item/neuFarm/seed/tangerine
+
+/datum/supply_pack/seeds/strawberry
+	name = "Strawberry"
+	cost = 15
```

Comment:
this should probably be way more expensive . at the least pear level

Reply:
remember soilson only need a single seed to start a mass production so . dont be too why with the prices